.github/workflows/daily.yml

```diff
@@ -7,38 +7,35 @@ on:
     - cron: "0 0 * * *"
 
 jobs:
+  provide_servers_json:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+
+      - name: Install Dependencies
+        run: yarn add ts-node axios
```

Comment:
you should add those deps to `skynet-js` package.json dev dependencies and commit them - `yarn add -D ts-node axios`

Then this step would be just `yarn`

Reply:
I don't want to install all the `skynet-js` dependencies in this step, it's overkill for the script. I could list the deps in a custom `package.json` but that doesn't seem necessary, either.